### PR TITLE
RELEASE (yay)

### PR DESCRIPTION
**Pull request by Seggan**
## Changes

Made it ready for release.
- Updated to RC-28
- Added recipes
- Added Automatic Door
- Increased protecting block range. As I tested gameplay, I realized 100 block range only takes care of the smallest rooms
- Added more sealable blocks
- Added Super Fan to increase range further
- Added Laserite
- Added Fusion Reactor
- A lot of bug fixes

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break world generation, mob spawning, and the like.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

File: src/main/java/io/github/addoncommunity/galactifun/api/items/spacesuit/SpaceSuit.java [hunk 1]
```diff
@@ -14,22 +14,22 @@
...
-import me.mrCookieSlime.Slimefun.cscorelib2.chat.ChatColors;
+import io.github.thebusybiscuit.slimefun4.libraries.dough.common.ChatColors;
 
 @ParametersAreNonnullByDefault
 public class SpaceSuit extends SlimefunItem implements ProtectiveArmor {
 
     private static final Map<String, SpaceSuit> SPACE_SUITS = new HashMap<>();
-    private static final NamespacedKey SPACE_SUIT_KEY = Galactifun.instance().getKey("space_suit");
-    private static final NamespacedKey OXYGEN_KEY = Galactifun.instance().getKey("oxygen");
+    private static final NamespacedKey SPACE_SUIT_KEY = AbstractAddon.createKey("space_suit");
```
Comment (Mooy1) [inline src/main/java/io/github/addoncommunity/galactifun/api/items/spacesuit/SpaceSuit.java:31]: ur meant to do Galactifun.createKey()

File: src/main/java/io/github/addoncommunity/galactifun/util/BlockPositionSet.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import java.util.Set;
+
+import javax.annotation.Nonnull;
+
+import io.github.thebusybiscuit.slimefun4.libraries.dough.blocks.BlockPosition;
+
+/**
+ * Utility class to avoid parameterized types in the {@link PersistentBlockPositions}
+ */
+public final class BlockPositionSet implements Set<BlockPosition> {
```
Comment (Mooy1) [inline src/main/java/io/github/addoncommunity/galactifun/util/BlockPositionSet.java:15]: You can do some unsafe casting instead to bypass that. iirc just do (Class) Set.class

**Review by Mooy1 — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/addoncommunity/galactifun/util/BlockPositionSet.java (hunk 2 above)
Comment (Seggan) [reply src/main/java/io/github/addoncommunity/galactifun/util/BlockPositionSet.java:15]: See Discord

**Review by Mooy1 — APPROVED** [review]
woop

**Pull request merged**